gateways/class-wc-ebanx-banking-ticket-gateway.php

```diff
@@ -40,8 +44,15 @@ public function __construct() {
 	 */
 	public function is_available() {
 		$country = $this->get_transaction_address( 'country' );
```

Comment:
It seems that $country attr contains a country ISO because later it's passed as an argument to a function called fromIso($contry), maybe changing its name would be a little less confusing. This goes for all is_available methods

Reply:
The name used is both, exactly the same as WooCommerce uses and exactly the same as used in `WC_EBANX_Constants::COUNTRY_ECUADOR` or `WC_EBANX_Constants::COUNTRY_BRAZIL`. The name was chosen to avoid confusion.